spec/cart_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'cart'
+require 'dish'
+require 'viewer'
+require 'menu'
+
```

Comment:
This is fine, in that it works, but it would be more idiomatic to use `let` statements